submissions/amooChristianah/javascript/productlist.js

```diff
@@ -19,4 +19,24 @@ const productList = [
 
 productList.forEach((product) => {
     console.log(product.productName + ' with '+ product.imageUrl + ' is sold for ' + product.productPrice)
-  });
\ No newline at end of file
+  });
+
+  /* TASK 2 */
+
+  var x = 123 - 7;
+
+
+for(var i = 1; i < x; i++){
```

Comment:
Since you are counting from '1`. I think your condition should be `<=` not `<`. Except maybe you want to count from `0`

Reply:
i have made changes...kindly review my PR now